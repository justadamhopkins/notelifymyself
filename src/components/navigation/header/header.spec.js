import React from 'react'
import { shallow } from 'enzyme'
import { Header } from '.'

describe('<Header/>', () => {
  it('Should render component', () => {
    const wrapper = shallow(<Header/>)
    expect(wrapper).toMatchSnapshot()
  })
})
