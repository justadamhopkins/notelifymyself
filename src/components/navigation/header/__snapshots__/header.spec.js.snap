// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header/> Should render component 1`] = `
<header
  className="Header"
>
  <Hamburger />
</header>
`;
